research/owaspMobileVerificationStandard.adoc

```diff
@@ -0,0 +1,145 @@
+= OWASP Mobile Verification Standard
```

Comment:
I think it might be better if we can add the link of the original source to this document. wdyt?